ifrs17/Import/4ImportScope-TechnicalMargin.ipynb

```diff
@@ -233,7 +233,10 @@
                 "\n    ",
                 "\n    [NotVisible] string EconomicBasis => EconomicBases.L;",
                 "\n    ",
-                "\n    double Value => GetScope<ValidAmountType>(Identity.DataNode).BeAmountTypes",
+                "\n    [IdentityProperty][NotVisible][Dimension(typeof(EstimateType))]",
+                "\n    string EstimateType => EstimateTypes.TM;",
+                "\n    ",
+                "\n    double Value => GetScope<ValidAmountType>((Identity.DataNode, EstimateType)).BeAmountTypes",
                 "\n                       .Except(GetStorage().GetNonAttributableAmountType())",
```

Comment:
You mean instead of making the ValidAmountType scope EstimateType sensitive to leave it unchanged and then "replace" from the BeAmountTypes the CDR AmountType with the CDRInitial (using this double negation logic inside the Except(get**Non**InitialCdr))?

Reply:
Yes, exactly.